From 50568b428edf6f41c396e4f49be7834e24d7cc2c Mon Sep 17 00:00:00 2001
From: Subramanian Srinivasan <subrsrin@codeaurora.org>
Date: Sat, 2 Mar 2019 15:14:06 -0800
Subject: [PATCH 07/10] Add scan filter for Transport Discovery data

Add scan filter support for Transport Discovery
data.

CRs-Fixed: 2418423
Change-Id: Ie2207554c141d1c673d367b0b010bf4beb9152a0
---
 include/hardware/bt_common_types.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/hardware/bt_common_types.h b/include/hardware/bt_common_types.h
index 0a315feda..34c707835 100644
--- a/include/hardware/bt_common_types.h
+++ b/include/hardware/bt_common_types.h
@@ -99,6 +99,9 @@ struct ApcfCommand {
   uint16_t company_mask;
   std::vector<uint8_t> data;
   std::vector<uint8_t> data_mask;
+  uint8_t org_id;
+  uint8_t tds_flags;
+  uint8_t tds_flags_mask;
 };
 
 #endif /* ANDROID_INCLUDE_BT_COMMON_TYPES_H */
-- 
2.28.0

